Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
 def customiseDataRun2Common_25ns(process):
     process = customiseDataRun2Common(process)
 
+    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
+    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
```

Comment:
Slava, could you just give your option (example)? 
I have only complied and run 'runTheMatrix --useInput all &' 
 which obviously don't act on 25ns data.